### DISCUSSION TITLE
Padding classes don't exist

### DISCUSSION THREAD

**Discussion by egermani** [tailwindlabs/tailwindcss#2953] [Help] [answered]
Hi all,

I've inherited a project that uses Tailwind, and I've found it very difficult to work with. I can work around most of the issues I've come across, but haven't been able to figure this one out:

pb-14 doesn't exist.

Per the [docs](https://tailwindcss.com/docs/padding), this should put 3.5rem of padding below an object. But it seems like this class has been stripped out of the stylesheet -- or not generated in the first place?

I assumed that this purgecss dependency is part of the problem. I disabled it in `tailwind.config.js` like so:

```
module.exports = {
  ...
  purge: {
      enabled: false,
  },
};
```

Does that not do anything?

After that didn't work, I also confirmed that I'm running my webpack process in development mode, so the purgecss plugin shouldn't be loaded at all. This is from the `postcss.config.js` file:

```
module.exports = {
  plugins: [
    require("tailwindcss"),
    require("autoprefixer"),
    ...(process.env.NODE_ENV === "production" ? [purgecss] : [])
  ]
};
```

My initial attempt at using `pb-14` was to drop it right in the HTML. I then tried `@apply pb-14` in a stylesheet, and that generated this error:

```
`@apply` cannot be used with `.pb-14` because `.pb-14` either cannot be found, or its actual definition includes a pseudo-selector like :hover, :active, etc. If you're sure that `.pb-14` exists, make sure that any `@import` statements are being properly processed *before* Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.
```

I don't know what it means for Tailwind CSS to "see" my CSS, or what a CSS tree is in this context, but perhaps if I can fix that I'll be in the clear.

So: how can I gain access to all of tailwind's padding classes? Thank you for any advice!

**Comment by MartinHughes-BPC** [comment, marked as answer]
Hi @egermani 

Do you know which version of Tailwind you're using? I believe 14 is a new value on the scale in (the recently released) Tailwind 2.0, so if you're running an older version, it won't work by default.

Cheers,
Martin.

**Reply by egermani** [reply]
Fantastic, that was the problem! Thank you, Martin, you've saved me a bunch of time.
